### PR TITLE
gtk+3: upstream Linux fixes

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
I think `at-spi2-atk` has bot been ported to core.

File: Formula/gtk+3.rb [hunk 1]
```diff
@@ -33,15 +33,28 @@ class Gtkx3 < Formula
 
   uses_from_macos "libxslt" => :build # for xsltproc
 
+  on_linux do
+    depends_on "cmake"
```
Comment (carlocab) [inline Formula/gtk+3.rb:37]: Not a build dependency?

Comment (danielnachun) [reply Formula/gtk+3.rb:37]: Yeah, this should only be a build dependency.

Comment (carlocab) [reply Formula/gtk+3.rb:37]: I also still find it bizarre that you can build with `meson` but somehow still need `cmake`. But I know it can happen...

**Comment by danielnachun** [comment]
> I think `at-spi2-atk` has bot been ported to core.

It's not, and now I realize I did this out of order because `at-spi2-atk` needs `dbus` to be upstreamed, which is still a WIP.  Should I close this or just tag do not merge until I upstream `at-spi2-atk`?

**Comment by danielnachun** [comment]
The final thing missing for this `at-spi2-atk`, which I will add as another new formula now that we have `at-spi2-core` added.

**Comment by danielnachun** [comment]
This is ready for review!

**Review by iMichka — APPROVED** [review]

**Pull request merged**